CONTRIBUTING.md

```diff
@@ -60,28 +60,40 @@ We require that your commit messages match our template. The easiest way to do t
 
     git config commit.template dev/git.commit.template
 
-### Run `./scripts/soundness.sh`
+### Run CI checks locally
```

Comment:
Instead of putting this everywhere we can now link to https://github.com/swiftlang/github-workflows?tab=readme-ov-file#running-workflows-locally